yoti_test.go

```diff
@@ -262,10 +260,13 @@ func TestYotiClient_ParseWithoutProfile_Success(t *testing.T) {
 				},
 			},
 		}
+		var err error
+		client.Key, err = loadRsaKey(key)
+		assert.NilError(t, err)
 
-		userProfile, activityDetails, err := client.getActivityDetails(encryptedToken)
+		userProfile, activityDetails, errStrings := client.getActivityDetails(encryptedToken)
```

Comment:
Don't think the `errStrings` changes were meant to be included, right?

Reply:
I'll have to manually fix the merge conflict one way or the other.